sources/packages/web/src/components/aest/students/assessment/AwardTable.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <v-table class="bordered">
+    <thead>
+      <tr>
+        <th scope="col" class="text-left">Loan/grant type</th>
+        <th scope="col" class="text-left">Estimated award</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Canada Student Loan</td>
+        <td>
+          {{ awardDetails[`${identifier}cslf`] ?? NOT_ELIGIBLE }}
```

Comment:
This code can be changed to a method that can receive a parameter like `cslf`.

Reply:
Makes sense.